ksp_plugin_test/plugin_test.cpp

```diff
@@ -186,6 +186,7 @@ class PluginTest : public testing::Test {
         plugin_(make_not_null_unique<TestablePlugin>(
                     initial_time_,
                     planetarium_rotation_)) {
+    google::LogToStderr();
```

Comment:
Is this needed here?  Do we actually log a real ephemeris anywhere, or do we always go through the mock?